src/main/kotlin/lotto/domain/LottoRanking.kt

```diff
@@ -0,0 +1,17 @@
+package lotto.domain
+
+enum class LottoRanking(val correctCount: Int, val price: Int) {
+    FIRST(6, 2000000000),
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000),
```
코틀린에서는 천단위로 언더바를 넣음으로써 가독성을 챙길 수 있습니다. 😄

Reply:
@DoLiLu 
코맨트가 10개가 넘어가서 축약된 코맨트들을 놓치신 것 같습니다. 🙏